Makefile

```diff
@@ -57,7 +57,10 @@ repl:
 
 test:
 	dune build @src/fmt --auto-promote || true
-	dune runtest || true
+	dune exec src/hazeltest/hazeltest.exe -- --regression-dir src/hazeltest/regressions
```

Comment:
should this be part of this PR, or should we pull it out and refactor with the tezt pr?

Reply:
This code was inherited directly from the tezt pr, so we can just undo it. There may be some unit tests specific to type aliases / tyvars. If so, we can park them on a separate branch and then later either fold into tezt pr or make a new one.

Reply:
fixed by 9a8f1049c